view/rooms/utils/page.js

```diff
@@ -75,4 +76,73 @@ module.exports = {
       console.log('Cannot inject dynamic contents into iframe.');
     }
   },
+
+  /**
+   * Checks whether a layout object is valid for an array of Bots
+   * @param {Object} layout - Layout object
+   * @param {Array} bots - Bots in the room
+   * @returns {boolean}
+   */
+  botLayoutIsValid(layout, bots) {
+    // Layout does not contain required field
+    if (!layout || !layout.leftColumn || !layout.middleColumn ||
```

Comment:
sometimes is better having a boolean function with a meaningful name (isValidLayout(layout)) instead of the comment to explain the long IF.

Or: if (layout.isvalid()) being more OO.